nodedb.go

```diff
@@ -535,28 +548,42 @@ func (ndb *nodeDB) cacheNode(node *Node) {
 }
 
 // Write to disk and memDB
-func (ndb *nodeDB) Commit() {
+func (ndb *nodeDB) Commit() error {
 	ndb.mtx.Lock()
 	defer ndb.mtx.Unlock()
 
+	var err error
 	if ndb.opts.KeepEvery != 0 {
 		if ndb.opts.Sync {
-			ndb.snapshotBatch.WriteSync()
+			err = ndb.snapshotBatch.WriteSync()
+			if err != nil {
+				return err
```

Comment:
Can these errors be wrapped with info on which DB failed?

Reply:
yes will wrap them